pkg/kubectl/proxy_server.go

```diff
@@ -64,7 +63,9 @@ func newProxyServer(target *url.URL) *ProxyServer {
 		req.URL.Host = target.Host
 		req.URL.Path = singleJoiningSlash(target.Path, req.URL.Path)
 	}
-	return &ProxyServer{ReverseProxy: httputil.ReverseProxy{Director: director}}
+	return &ProxyServer{
+		ReverseProxy: httputil.ReverseProxy{Director: director},
```

Comment:
I think this is more readable, but most Go I've seen would keep this on one line. Or is the OSS style different?